From 8a31083fbc4d684588ba04dd02e1f00d335227c7 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Mon, 15 Jan 2018 17:53:31 -0800
Subject: [PATCH] TRITON-47 CloudAPI ListVolumes documentation should mention
 default behavior wrt state filtering Reviewed by: Marsell Kukuljevic
 <marsell@joyent.com> Approved by: Marsell Kukuljevic <marsell@joyent.com>

---
 docs/index.md | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/docs/index.md b/docs/index.md
index 1f15fa6..7f55c9c 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -9082,6 +9082,9 @@ a VM).
 _Available only for CloudAPI services running in datacenters for which NFS
 volumes support has been explicitly enabled. By default, it is disabled._
 
+By default (if the request doesn't include the `state` and `predicate`
+parameters), volumes in state === `'failed'` are not included in the response.
+
 ### Input
 
 Param       | Type         | Description
@@ -9090,7 +9093,7 @@ name        | String       | Allows filtering volumes by name
 predicate   | String       | URL-encoded JSON string representing an object that can be used to build a LDAP filter. This LDAP filter can search for volumes on arbitrary indexed properties. More details below
 size        | String       | Allows filtering volumes by size, e.g `size=10240`
 state       | String       | Allows filtering volumes by state, e.g `state=failed`
-| type      | String       | Allows filtering volumes by type, e.g `tritonnfs`
+type        | String       | Allows filtering volumes by type, e.g `tritonnfs`
 
 #### Searching by name
 
-- 
2.21.0

